packages/host/tests/acceptance/catalog-app-test.gts

```diff
@@ -447,77 +447,75 @@ module('Acceptance | Catalog | catalog app tests', function (hooks) {
 
-        // Step 3: Verify we're in catalog grid / list view (not showcase view)
-        assert
-          .dom('[data-test-showcase-view]')
-          .doesNotExist('Should be in list view after applying filter');
+        test('switch to apps tab', async function (assert) {
+          await click('[data-tab-label="Apps"]');
+          assert
+            .dom('[data-test-navigation-reset-button="app"]')
+            .exists(`"All Apps" button should exist`)
+            .hasClass('is-selected');
+          assert.dom('[data-test-boxel-radio-option-id="grid"]').exists();
+        });
 
-        //step 4: click on the home button
-        await click('[data-test-navigation-reset-button="showcase"]');
+        test('switch to cards tab', async function (assert) {
+          await click('[data-tab-label="Cards"]');
+          assert
+            .dom('[data-test-navigation-reset-button="card"]')
+            .exists(`"All Cards" button should exist`)
+            .hasClass('is-selected');
+          assert.dom('[data-test-boxel-radio-option-id="grid"]').exists();
+        });
 
-        //step 5: Verify you looking at showcase view
-        assert
-          .dom('[data-test-showcase-view]')
-          .exists('Should return to showcase view after clicking Catalog Home');
+        test('switch to fields tab', async function (assert) {
+          await click('[data-tab-label="Fields"]');
+          assert
+            .dom('[data-test-navigation-reset-button="field"]')
+            .exists(`"All Fields" button should exist`)
+            .hasClass('is-selected');
+          assert.dom('[data-test-boxel-radio-option-id="grid"]').exists();
+        });
 
-        // Step 6: Verify filters are reset - make sure you reset category/tag/search state
-        assert
-          .dom('[data-test-filter-search-input]')
-          .hasValue('', 'Search input should be cleared');
-        // Category and tag filters should be unselected
-        assert
-          .dom('[data-test-filter-list-item].is-selected')
-          .doesNotExist('No category should be selected after reset');
-        assert
-          .dom('[data-test-tag-list-pill].selected')
-          .doesNotExist('No tag should be selected after reset');
+        test('switch to skills tab', async function (assert) {
+          await click('[data-tab-label="Skills"]');
+          assert
+            .dom('[data-test-navigation-reset-button="skill"]')
+            .exists(`"All Skills" button should exist`)
+            .hasClass('is-selected');
+          assert.dom('[data-test-boxel-radio-option-id="grid"]').exists();
+        });
       });
-    });
 
-    module(
-      'other tab is active, showcase tab is NOT active',
-      async function () {
-        test('filters reset when clicking on the "All Apps" button', async function (assert) {
-          // Step 1: Switch to Apps tab
-          await click('[data-tab-label="Apps"]');
+      module('showcase tab is active', async function () {
```

Comment:
why is this inside the tab navigation block? This particular block is checking that the filters is reset when we click the "Catalog Home" or "All Apps" button